client/my-sites/email/index.js

```diff
@@ -29,16 +30,17 @@ export default function () {
 		handlers: [ ...commonHandlers, controller.emailManagement, makeLayout, clientRender ],
 	} );
 
+	const productType = `:productType(${ GOOGLE_WORKSPACE_PRODUCT_TYPE }|${ GSUITE_PRODUCT_TYPE })`;
```

Comment:
TiL! How did you find that? The page.js docs only have a param wildcard example which is something completely different...

Reply:
Yes, it wasn't obvious to me as well. Since `Page.js` documentation mentioned:

> The router uses the same string-to-regexp conversion that Express does, so things like ":id", ":id?", and "*" work as you might expect.

I checked the `Express.js` documentation, and finally stumbled upon [this](https://expressjs.com/en/5x/api.html#path-examples) and [this](https://expressjs.com/en/guide/routing.html#route-paths).